server/src/test/java/org/elasticsearch/index/query/FieldMaskingSpanQueryBuilderTests.java

```diff
@@ -73,13 +74,13 @@ public void testFromJson() throws IOException {
                 "}";
         Exception exception = expectThrows(ParsingException.class, () -> parseQuery(json));
         assertThat(exception.getMessage(),
-            equalTo("field_masking_span [query] as a nested span clause can't have non-default boost value [0.23]"));
+            equalTo(NAME.getPreferredName() + " [query] as a nested span clause can't have non-default boost value [0.23]"));
     }
 
```

Comment:
Can we also add a test that uses the old deprecated name, and checks for a deprecation warning? There's an `assertWarnings()` helper method that will check for specific deprecation strings.

Reply:
Sure, i'll add a new test in my next commit.